packages/devops/ci/testing.env.encrypted

```diff
@@ -1,2 +1,2 @@
 codeship:v2
-Xe0EbawQHSultl2ATorXCdSl5h/yWZie0B28nlSLBSl9NJ5VNFo0QTq146p/LjJagfL0+My5ZGTHfJ3vVf6E9+2eo6xn/z59xK+RtcD61w7uFup2Z5ocqTez5aXK6K2qZbyWmdSyY1cDyXR9gsQWtUU9rakh7SyQgxZRwez8ZfU2SdS+Iglpfc/77SNzYVGMnwJb27/h93Bb+qvXeOsgbG+J05/1FEoFD3Xn3KIZmC6nZFWHMc0SHKjVLkrrTXnxrGQvl7VocY7dWC4K80sH7hnGhVDpA2e3v5K9BqDQv7N0EgiGqvueuugId+l1rlTCKQ4+9LBK18V3cU9em6DG1/7dB9sReCJSa5LEPMStbPQQ7ubgx2G2vRIL3Pci/PWmyBkGDfNy417PrTMpdln+Wnmy96jQ0+N8yEo1cwmu/L2WUs2NTyNkLVibVlhL6q2YpwuNjTqNJwzPV/7pmXHPdSfybmW+l5gD5y53jZtnJa8VC4VsxOQS4Q9HUfry69nTVK59BkVTaRRR1/FmPVmIDUkxTp4crwG0Y6xU4+o073yYZ8TXWemeCY64jXeOUWECcbltKSZKqr8102Gbt/trwSo3Kh8bY2rZPYpp/otlk9/9HIQYAP6zjaX19owNXPtlpFGSkH1sXLCZcMrVT4q5MFZ6qv5A4fpeptWqog1ZrRy44JEGSuOsERrBDgfPJESKM6Aul5A+uwW5DuvJ60aVhNSs3OYApPcx1XdbXEMqup1IdTtfgsE0AK3WlLcHLmyyeNMcbBHGlzme4Gvbo0NQpwK3VXbfprB4AjOigFirV2ThprBAnPO+7cs6DYkgHPKBgfNs7pDrFSER5pWD
\ No newline at end of file
+xnpR6UV/QIrB5eKb7b0329V76cFEnWFSc8M9eEOPaYwdhW1TF20VKtu58Kj1LMzuaXGuZDr/jjIRBcLyfwkmS9kZSPC8+bEvf3aqrW1PSYLmqZI8jYSgxrcsvP/6/YoO9JEsxg0pu/WqkiXasPuY7DFXtVU3E9aBH4iFHpMV4FUIq2wEaQdJDsOkqIkpP2jSsgaWn6iKd7kViVAhGM930pNECmSeStYJ20TJS/7KPumHO5Jg5KbWIMtRWonPAMclI2/Z001cy0uIvy5zHRQ6gpS9eVtsv4m7mVwWVauBF5ZAPezNh7lzpLM/UxGNakO10z0pr7BZ6s6Bg8a7+mHvAv17QZgGOwaPmEZ92Y0zMALdwNYckJ9qnRaI0inOszWK6h1tIrIDvZM2kR+S63qO25bZgn4jajXmFeTJtHkPda9H2eh7Shd8wkatkfaQD+/Bngf3DYxfkUENC1z+xvrsss/gqEPD+lZpUhLUa9PbqqDuMPLxp2k/9DOn+qYXQVZB7bAZVpd+Kv8iyggyLl6uYln6iRJJJEIsHsS41ReHrYnOBDg3DXEAPHBGpnP8/gljtMkBA/H2aF3JZdItPU3auJAdDO4r0bTPK3iHNXftgsEew3I2+lNGKSKC65rfQ36yatXLoR06y+hz9/+HkZETn9L+oQoLTDXb91a/4dn5stRFYuhTNdJ6FpvhRGLnpVHEBOtWhdNjNnPYcs1UEUWeOjPkfthmV2O+NeLN+UUObA0D3qKNs3A/4M3DEwHOAWe5+0xC/jhCgGNBHUwOfmXxM07de5wYty2ieHkzHsZ6kZ9IEkT43Pck+hh7wJ5lLc3xP5L235Tx1pPBDN3uSESBH7U=
```

Comment:
Changed db name from `tupaia` to `tupaia_test`. Not sure how this was working before... e.g. entity-server's tests explicitly run against a database named `tupaia_test`.

Reply:
Oh yeah I mean I'm not sure how it was creating the db with name tupaia_test before.

https://github.com/beyondessential/tupaia/blob/dev/packages/database/scripts/setupTestDatabase.sh#L16
Reading the script it should have been creating a db called tupaia not tupaia_test. Oh well, it's not important, just a curiosity.